include/auth.php

```diff
@@ -64,25 +72,29 @@
 		}
 	}
 
-	if (x($_SESSION,'uid')) {
-
+	if (x($_SESSION, 'uid')) {
 		// already logged in user returning
-
-		$check = Config::get('system','paranoia');
+		$check = Config::get('system', 'paranoia');
 		// extra paranoia - if the IP changed, log them out
 		if ($check && ($_SESSION['addr'] != $_SERVER['REMOTE_ADDR'])) {
-			logger('Session address changed. Paranoid setting in effect, blocking session. '.
-				$_SESSION['addr'].' != '.$_SERVER['REMOTE_ADDR']);
+			logger('Session address changed. Paranoid setting in effect, blocking session. ' .
+				$_SESSION['addr'] . ' != ' . $_SERVER['REMOTE_ADDR']);
 			nuke_session();
 			goaway(System::baseUrl());
 		}
 
-		$r = q("SELECT `user`.*, `user`.`pubkey` as `upubkey`, `user`.`prvkey` as `uprvkey`
-		FROM `user` WHERE `uid` = %d AND NOT `blocked` AND NOT `account_expired` AND NOT `account_removed` AND `verified` LIMIT 1",
-			intval($_SESSION['uid'])
+		$user = dba::select('user',
+			[],
+			[
+				'uid' => intval($_SESSION['uid']),
```

Comment:
Same like above. Good thing with prepared statements is that they care internally about this stuff.